core/tools/generate_names.cc

```diff
@@ -81,6 +81,7 @@ NameDef names[] = {
     {"cfgAlias", "<cfgAlias>"},
     {"magic", "<magic>"},
     {"unconditional", "<unconditional>"},
+    {"argPresent", "<argPresent>"},
```

Comment:
Should this be `argPresentTemp`?

Reply:
It could be, though it's not used as a temporary.

Reply:
What do you mean? The only place where I see `Names::argPresent` used is to create a temporary:

https://github.com/sorbet/sorbet/pull/3366/files#diff-8b024831e1dcec03148f822d2f42ee3fR75

Reply:
I meant that the value itself is used, and not just an intermediate for assigning to something else. In this case, the `present` variable is used as the choice for which branch to take when exiting the block.